docs/03-tutorials/00-application-connectivity/ac-04-register-secured-api.md

```diff
@@ -118,6 +118,44 @@ To create such a Secret, run this command:
 ```bash
 kubectl create secret generic {SECRET_NAME} --from-literal clientId={CLIENT_ID} --from-literal clientSecret={CLIENT_SECRET} -n kyma-integration
 ```
+## Register an OAuth-secured API with token obtained from client certificate-secured endpoint
+
+This is an example of the **service** object for an API secured with OAuth, where the token is fetched from Mutual-TLS secured endpoint:
+
+```yaml
+  - id: {TARGET_UUID}
+    name: my-oauth-service
+    displayName: "My MTLS OAuth Service"    
```

Comment:
```suggestion
    name: my-mTLS-oauth-service
    displayName: "My mTLS OAuth Service"    
```

Reply:
Copy & Paste error. Please feel free to change it